streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -269,6 +270,39 @@ class StreamingContext private[streaming] (
     RDDOperationScope.withScope(sc, name, allowNesting = false, ignoreParent = false)(body)
   }
 
+  private[streaming] val recoverableAccuNameToId: mutable.Map[String, Long] = {
```

Comment:
All `Accumulator[_,_]` has been restored in `Accumulators.original` map, I don't want to duplicate them in memory. There are two disadvantage: 1. extra effort to keep consistency between two maps (2) extra memory space